projects/upb/project.yaml

```diff
@@ -1,7 +1,24 @@
 homepage: "https://github.com/protocolbuffers/upb"
 language: c++
 primary_contact: "haberman@google.com"
-
+auto_ccs:
```

Comment:
Do we typically do this with individual addresses? Can we use a group? Or is this required by the setup?
Individual addresses are brittle if the team adds/removes members.

Reply:
These are good points but unfortunately we have no support for groups.

Reply:
Yeah, that's why we're reliant on the redundancy of protobuf-oss-fuzz@ (routing for triage) and individual addresses (for actual access). Will rely on documentation to ensure we keep this up-to-date.

Reply:
Oh... but protobuf-oss-fuzz@google.com is a group, no?

Reply:
It will get emails but members won't be allowed to login without being added to auto_ccs.